web/src/utils/dashboard/colorPalette.ts

```
@@ -0,0 +1,284 @@
+  });
+  return [min, max];
+};
+
+export const getSQLMinMaxValue = (yaxiskeys: any, searchQueryData: any) => {
+  // need min and max value for color
+  let min = Infinity;
+  let max = -Infinity;
+
+  searchQueryData[0]?.forEach((data: any) => {
+    yaxiskeys?.forEach((key: any) => {
+      if (data[key] !== undefined && !isNaN(data[key]) && data[key] !== null) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)